zerver/views/auth.py

```diff
@@ -843,11 +855,12 @@ def done(self, form_list: List[Form], **kwargs: Any) -> HttpResponse:
             return super().done(form_list, **kwargs)
 
 
-@has_request_variables
+@typed_endpoint
 def login_page(
     request: HttpRequest,
     /,
-    next: str = REQ(default="/"),
+    *,
```

Comment:
Shouldn't the `/,` in the line above be deleted?

Reply:
Well without `/`  `mypy` complains with the following error : 
```
 error: Argument 1 to "typed_endpoint" has incompatible type "Callable[[Arg(HttpRequest, 'request'), DefaultNamedArg(str, 'next'), KwArg(Any)], HttpResponse]"; expected "Callable[[HttpRequest, DefaultNamedArg(str, 'next'), KwArg(Any)], HttpResponse]"  [arg-type]
```
and suggests that `request` should be marked as positional-only, this only happens when `**kwargs` is used

Reply:
`request` being positional-only seems harmless, so this is probably fine.